core/arch/arm/plat-rcar/conf.mk

```diff
@@ -8,6 +8,7 @@ $(call force,CFG_SECURE_TIME_SOURCE_CNTPCT,y)
 $(call force,CFG_WITH_ARM_TRUSTED_FW,y)
 $(call force,CFG_SCIF,y)
 $(call force,CFG_CORE_LARGE_PHYS_ADDR,y)
+$(call force,CFG_CORE_ARM64_PA_BITS,36)
```

Comment:
Does LPAE mandate 36 bits? If so, this line could preferably be moved to some common place.

Reply:
No, LPAE is fine with 32 bits too.